odl/space/numpy_ntuples.py

```diff
@@ -145,27 +145,38 @@ def one(self):
 
         Examples
         --------
-        >>> c3 = Cn(3)
+        >>> c3 = NumpyNtuples(3, dtype=complex)
         >>> x = c3.one()
         >>> x
-        Cn(3).element([(1+0j), (1+0j), (1+0j)])
+        ntuples(3, 'complex').element([(1+0j), (1+0j), (1+0j)])
         """
         return self.element(np.ones(self.size, dtype=self.dtype))
 
+    def __repr__(self):
+        """Return ``repr(self)``."""
+        class_name = 'ntuples'
```

Comment:
That name is a bit misleading (always was). `constructor_name` would be more precise.